pkg/controller/keyvault/ingress_secret_provider_class.go

```diff
@@ -22,9 +23,9 @@ import (
 	secv1 "sigs.k8s.io/secrets-store-csi-driver/apis/v1"
 )
 
-var (
-	ingressSecretProviderControllerName = controllername.New("keyvault", "ingress", "secret", "provider")
-)
+const nginxAnnotationPrefix = "nginx.ingress.kubernetes.io/"
```

Comment:
💡 Observation share: You might know better design, but is this common occurrences or just one case? 

if its just used in one place then I guess move it to line-169, but if this will grow, might be worth to think it upfront for wider design for some sort of structure in place.

Reply:
this will only be used in this place for the foreseeable future